sklearn/decomposition/_dict_learning.py

```diff
@@ -1835,6 +1827,11 @@ class MiniBatchDictionaryLearning(_BaseSparseCoding, BaseEstimator):
         - `'threshold'`: squashes to zero all coefficients less than alpha from
           the projection ``dictionary * X'``.
 
+        If `None`, defaults to `alpha`.
+
+        .. versionchanged:: 1.2
+            When None, default value changed from 1.0 to `alpha`.
```

Comment:
I don't quite understand this. The default of `transform_algorithm` is "omp" and other valid values are strings. So how/when would a user pass `None` here? An even when they do, from the code it looks like that `transform_alpha` will be set to 1., not `transform_algorithm` (which this part of the doc string is about, I think)

Reply:
I put that in the wrong param description. It should have been in ``transform_alpha``. Fixed, thanks.